src/antsibull_changelog/config.py

```diff
@@ -416,6 +564,7 @@ class ChangelogConfig(p.BaseModel):
         "alphanumerical",
     ] = "alphanumerical"
     vcs: t.Literal["none", "auto", "git"] = "none"
+    output: list[ChangelogOutput]
```

Comment:
```suggestion
    output: p.conlist(ChangelogOutput, min_length=1)
```

and can remove the validator

Reply:
`p.conlist` didn't work, I used the solution from https://stackoverflow.com/a/78042465 instead. I also added an explicit dependency on `annotated_types` to avoid problems if pydantic eventually removes that dependency.